2009/4xxx/GSD-2009-4123.json

```diff
@@ -53,6 +53,46 @@
                     "uuid": "f7fde20b-d4b6-4f94-934b-6ddebf4193c6"
                 }
             ]
+        },
+        "github.com/rubysec/ruby-advisory-db": {
+            "gem": "jruby-openssl",
+            "platform": "jruby",
+            "cve": "2009-4123",
+            "url": "http://jruby.org/2009/12/07/vulnerability-in-jruby-openssl",
+            "title": "jruby-openssl Gem for JRuby fails to do proper certificate validation",
+            "date": "2009-12-07",
+            "description": "A security problem involving peer certificate verification was found where\nfailed verification silently did nothing, making affected applications\nvulnerable to attackers. Attackers could lead a client application to believe\nthat a secure connection to a rogue SSL server is legitimate. Attackers could\nalso penetrate client-validated SSL server applications with a dummy\ncertificate.\n",
+            "patched_versions": [
+                ">= 0.6"
+            ]
+        }
+    },
+    "gsd": {
+        "osvSchema": {
+            "schema_version": "1.4.0",
+            "id": "GSD-2009-4123",
+            "modified": "2009-12-07T00:00:00.000Z",
+            "published": "2009-12-07T00:00:00.000Z",
+            "aliases": [
+                "CVE-2009-4123"
+            ],
+            "summary": "jruby-openssl Gem for JRuby fails to do proper certificate validation",
+            "details": "A security problem involving peer certificate verification was found where\nfailed verification silently did nothing, making affected applications\nvulnerable to attackers. Attackers could lead a client application to believe\nthat a secure connection to a rogue SSL server is legitimate. Attackers could\nalso penetrate client-validated SSL server applications with a dummy\ncertificate.\n",
+            "affected": [
+                {
+                    "package": {
+                        "ecosystem": "RubyGems",
+                        "name": "jruby-openssl",
+                        "purl": "pkg:gem/jruby-openssl"
+                    }
+                }
+            ],
```

Comment:
Unfortunately I couldn't figure out how to sanely map the way ruby-advisory-db uses patched/unaffected into the introduced/fixed ranges OSV supports.

It's probably possible, and I'm just too sleep deprived atm to figure it out.